script.js

```diff
@@ -0,0 +1,173 @@
+let movieRegistry = [];
+
+const api_key = 'a3016ba6049f537e33cb6a0c27a8a821';
+const url_path = 'https://api.themoviedb.org/3/movie';
+const img_path = 'https://image.tmdb.org/t/p/w500/'
+const search_path = 'https://api.themoviedb.org/3/search/movie?api_key=a3016ba6049f537e33cb6a0c27a8a821&language=en-US&query=';
+const trailer_path = 'https://api.themoviedb.org/3/movie/';
+const trailer_path_pt2 = '/videos?api_key=a3016ba6049f537e33cb6a0c27a8a821&language=en-US'
+let page = 1;
+
+// Query Selectors
+
+const movieGridElement = document.getElementById('movies-grid');
+const loadMoreMoviesElement = document.getElementById('load-more-movies-btn');
+const searchFormElement = document.getElementById('search-form');
+const searchInputElement = document.getElementById('search-input');
+const closeSearchButtonElement = document.getElementById('close-search-btn');
+const movieInfoElement = document.getElementById('movie-info');
+const headerElement = document.getElementById('header');
+const closeMovieButtonElement = document.getElementById('close-movie-btn');
+
+// Initialize page
+
+async function getData() {
+    let response = await fetch(url_path + '/now_playing?api_key=' + api_key + '&page=' + page.toString());
```

Comment:
This is perfectly fine, but you could also use [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#string_interpolation) to make this a bit easier to read.

```suggestion
    let response = await fetch(`${url_path}/now_playing?api_key=${api_key}&page= ${page.toString()}`);
```